packages/http-server-js/generated-defs/package.json.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license.
+
+export const hsjsDependencies: Record<string, string> = {
+  "@types/express": "^5.0.1",
+  "@types/morgan": "^1.9.9",
+  "@types/node": "~22.13.11",
+  "@types/swagger-ui-express": "^4.1.8",
+  "@typespec/compiler": "workspace:^",
+  "@typespec/http": "workspace:^",
+  "@typespec/http-specs": "workspace:^",
+  "@typespec/internal-build-utils": "workspace:^",
+  "@typespec/openapi3": "workspace:^",
+  "@typespec/spector": "workspace:^",
+  "@typespec/tspd": "workspace:^",
+  "@vitest/coverage-v8": "^3.0.9",
+  "@vitest/ui": "^3.0.9",
+  "decimal.js": "^10.5.0",
+  "express": "^5.1.0",
+  "fs-extra": "^11.2.0",
+  "globby": "~14.1.0",
+  "inquirer": "^12.5.0",
+  "morgan": "^1.10.0",
+  "ora": "^8.1.1",
+  "p-limit": "^6.2.0",
+  "pathe": "^2.0.3",
+  "picocolors": "~1.1.1",
```

Comment:
Does this result in all of these dependencies getting installed when someone enables `express` scaffolding? I think a lot of these are just our own dev dependencies that a user wouldn't need.

Reply:
No, only the ones that the generated project actually imported. This little module is sort of a preprocessing step that puts all my own dependency versions in this file, then the scaffolding tool actually looks at what external dependencies were actually imported and injects those constraints into the scaffolded package.json.